src/transformers/generation/flax_utils.py

```diff
@@ -744,9 +786,11 @@ def _beam_search(
         max_length: Optional[int] = None,
         pad_token_id: Optional[int] = None,
         eos_token_id: Optional[int] = None,
+        output_scores: Optional[bool] = None,
         length_penalty: Optional[float] = None,
         early_stopping: Optional[Union[bool, str]] = None,
         logits_processor: Optional[FlaxLogitsProcessorList] = None,
+        num_beam_hyps_to_keep: Optional[int] = 1,
```

Comment:
Note that this should default be `None` otherwise we'll never accept `num_return_sequences` from the generation config